File: C/Prime_Number.c
//? The objective of this C program is to generate prime numbers in a given range..
#include <stdio.h>
int main()
{
    int start, end;
    printf("Enter start of range:");
    scanf("%d", &start);
    printf("\nEnter end of range:");
    scanf("%d", &end);
    printf("All prime numbers between %d and %d are:", start, end);
    while (start <= end)
    {
        if (start == 1)
        {
            start++;
            continue;
        }
        int temp = start, i = 2, flag = 0;
        while (i < temp)
        {
            if (temp % i == 0)
            {
                flag = 1;
                break;
            }
            i++;
        }
        if (flag == 0)
        {
            printf("\n%d", start);
        }
        start++;
    }
    return 0;
}

File: Java/PrimeNumber.java

//? The objective of this JaVa program is to generate prime numbers in a given range.
import java.util.Scanner;

public class PrimeNumber {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        System.out.println("ALl Prime Numbers will be generated in a given range!");
        int start;
        int end;
        System.out.println("Enter start of range:");
        start = sc.nextInt();
        System.out.println("Enter end of range:");
        end = sc.nextInt();
        System.out.println("All Prime Numbers between " + start + " and " + end + " are:-->");
        while (start <= end) {
            if (start == 1) {
                start++;
                continue;
            }
            int temp = start;
            int i = 2;
            int flag = 0;
            while (i < temp) {
                if (temp % i == 0) {
                    flag = 1;
                    break;
                }
                i++;
            }
            if (flag == 0) {
                System.out.println(start);
            }
            start++;
        }
        sc.close();
    }
}
